build.sh

```diff
@@ -3,17 +3,21 @@
 set -eux
 
 ICE_VERSION=${1:-"3.6.5"}
+BUILD=${TRAVIS_BRANCH:-dev}
```

Comment:
what is the rationale for adding travis-branch in the name?
When we tag, we will need to remember to pass a value, so we do not end with ``-dev`` in the name.
This should be indicated in the README

Reply:
I have not tested the tagging and name change during the tagging

Reply:
It was fixed in the last commit after you tested